src/components/books.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import '../App.css';
+
+const Books = () => (
+  <div className="books">
+    <ul>
+      <li>
+        Beyond Order
+        <span>Jordan Peterson</span>
+      </li>
+      <li>
+        Mpas of Meaninng
+        <span>Jordan Peterson</span>
+      </li>
+      <li>
+        12 Rules for Life
+        <span>Jordan Peterson</span>
+      </li>
+      <li>
+        Sapiens
+        <span>Yuval Harrari</span>
+      </li>
+    </ul>
+    <form>
+      <input type="text" placeholder="Book title..." />
+      <input type="text" placeholder="Book Author" />
+      <input type="submit" />
+    </form>
+  </div>
+);
+
+export default Books;
```

Comment:
[OPTIONAL] You did a great job working on your Home page! 🚀 However, I would suggest that you extract at least one reusable component from this, this is because this markup can get messy very quickly, and extracting meaningful components can make reading your code as well as handling events easier! 🤓 🔧

_💡 I can see three logical components here: Booklist, represented by the `<ul>`, a Book, represented by each` <li>`, and the NewBookForm, represented by the `<form>`!_

Reply:
Done.